src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj/project.pbxproj

```diff
@@ -3186,7 +3207,8 @@
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = "";
 				SWIFT_OBJC_BRIDGING_HEADER = "Other/TogglDesktop-Bridging-Header.h";
 				SWIFT_VERSION = 5.0;
-				SYSTEM_FRAMEWORK_SEARCH_PATHS = "$(inherited) $(SYSTEM_LIBRARY_DIR)/PrivateFrameworks";
+				SYSTEM_FRAMEWORK_SEARCH_PATHS = "$(SYSTEM_LIBRARY_DIR) $(SYSTEM_LIBRARY_DIR)/PrivateFrameworks/**";
```

Comment:
For Release mode, we have to add `$(SYSTEM_LIBRARY_DIR)` and `$(SYSTEM_LIBRARY_DIR)/PrivateFrameworks/**` (Recursive mode) to fix the Import issue. I'm not sure why the Debug and Release mode are different, but it works.

Able to
- Run on DEV mode
- Build on Production mode

Reply:
Unfortunately, this solution doesn't work anymore.

I encounter that failed build in TogglDesktop and TogglDesktop-Production scheme:
```
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.15.sdk/System/Library/Frameworks/AuthenticationServices.framework/Headers/AuthenticationServices.h:8:9: 'AuthenticationServices/ASFoundation.h' file not found
```
and
```
/Users/<username>/Documents/Work/Toggl/toggldesktop/src/ui/osx/TogglDesktop/Feature/AppleSignIn/AppleAuthenticationService.swift:10:8: Could not build Objective-C module 'AuthenticationServices'
```

-------------
From the preliminary investigation, it turns out that Xcode doesn't recognize the path of System Framework since we override the default value by `$(SYSTEM_LIBRARY_DIR)/PrivateFrameworks` (Private Touch Bar Framework).

I trial and error with various combination (with $(SYSTEM_LIBRARY_DIR), $inherited), but no luck. Additionally, the error only occurs in TogglDesktop (where we have Private framework), but work very well with AppStore scheme

Reply:
I suggest to remove the Touch Bar Private Framework from non-AppStore version in order to work with Apple Sign In.